x-pack/plugins/viz_editor/public/frame/main/droppable_pane.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import { VisModel } from '../../common';
+import { Draggable } from '../../common/components/draggable';
+import { VisualizationModal } from '../../common/components/visualization_modal';
+import { GetSuggestionsType, Suggestion } from '../../editor_plugin_registry';
+
+interface ModalProps {
+  visModel: VisModel;
+  children: any;
+  getAllSuggestionsForField: GetSuggestionsType<VisModel<any, any>>;
+  onChangeVisModel: (newState: VisModel) => void;
+}
+
+export function DroppablePane({
+  visModel,
+  getAllSuggestionsForField,
+  onChangeVisModel,
+  children,
+}: ModalProps) {
+  // tslint:disable-next-line:no-shadowed-variable
+  const initialState = {
+    isOpen: false,
+    suggestions: [] as Array<Suggestion<VisModel<any, any>>>,
```

Comment:
You shouldn't need to do more than `[] as Suggestion[]`

Reply:
Good. I hated that signature, but it was the one VS Code generated, and I didn't know you had a Suggestion definition.